internal/cli/deployment_create.go

```diff
@@ -128,10 +128,14 @@ func (c *DeploymentCreateCommand) Run(args []string) int {
 		app.UI.Output("")
 		switch {
 		case releaseUrl != "":
-			printInplaceInfo(inplace, app)
-			app.UI.Output("   Release URL: %s", releaseUrl, terminal.WithSuccessStyle())
-			app.UI.Output("Deployment URL: https://%s", deployUrl, terminal.WithSuccessStyle())
-
+			if deployUrl != "" {
```

Comment:
Thanks for making the PR @gunadhya and you're on the right track! We already have a [template](https://github.com/hashicorp/waypoint/blob/95ec521fe35a8b57df49d9e6e33404a57ee17a83/internal/cli/deployment_create.go#L210) defined when URL service is disabled, I would suggest refactoring the case switch statement to hit that code.

Reply:
Oh! Do you mean a new `case` like this?
```go
    case releaseUrl != "" && deployUrl == "":
	app.UI.Output(strings.TrimSpace(deployNoURL)+"\n", terminal.WithSuccessStyle())
	app.UI.Output("   Release URL: %s", releaseUrl, terminal.WithSuccessStyle())
```

Reply:
I meant a `case deployUrl == ""` that wouldn't print out the release URL, but thinking more about it we do want the release URL too, so an if statement inside the `releaseUrl` case was the proper approach, I'd do something like

```
case releaseUrl != "":
    printInplaceInfo(inplace,app)
    app.UI.Output(" Release URL: %s", releaseUrl, terminal.WithSuccessStyle())
    if deployUrl != "" {
        app.UI.Output("Deployment URL: https://%s", deployUrl 
    } else {
        app.UI.Output(strings.TrimSpace(deployNoURL)+"\n", terminal.WithSuccessStyle())
    }
```